projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/commandsAndMenu.tsx

```diff
@@ -52,8 +55,11 @@ function add_command(commands: CommandRegistry, schemaNaming: string, label: str
           runningVdkOperation = true;
           jobData.set(VdkOption.PATH, workingDirectory);
           await jobdDataRequest();
```

Comment:
why do you need to run this here?

Reply:
@murphp15 if there is a config file in the current directory it gathers the information about the job from it and sets default values to the inputs